Changelog.md

```diff
@@ -1044,6 +1044,7 @@ everything is set up.
 * RTL-language characters in usernames no longer overlay post dates [#2339](https://github.com/diaspora/diaspora/issues/2339)
 * Overflow info author mobile web. [#3983](https://github.com/diaspora/diaspora/issues/3983)
 * Overflow name author mobile post. [#3981](https://github.com/diaspora/diaspora/issues/3981)
+* Fix weird error message on too long tag. [#5737](https://github.com/diaspora/diaspora/issues/5737)
```

Comment:
Looks like you hit the wrong release here ;) Btw if you don't add a changelog entry, we'll do in the merge commit, so it's fine to leave that out. So I'd say just drop this commit: `git reset --hard 18a1958; git push -f origin 5737-too-long-tag-name`.